src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -142,7 +144,7 @@
     <Compile Include="TripleDesTests.cs" />
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Common\Interop\Unix\Interop.Libraries.cs</Link>
-    </Compile>
+    </Compile> 
```

Comment:
I'm a little confused by the trailing space addition here.  (It also feels like the whole edit to this file could be reverted, since it just moved a line up... though it did group it with its friends...)